package.json

```diff
@@ -14,7 +14,7 @@
     "lint-config": "eslint ./*.js",
     "lint-react": "eslint --ext .js,.jsx src",
     "lint-test": "eslint --ext .js,.jsx test",
-    "npm:preversion": "builder run lint && builder run build-static",
+    "npm:preversion": "builder run lint && --env='{"NODE_ENV":"production"}' builder run build-static",
```

Comment:
@paulathevalley -- Actually hook in the changed archetype and in a project run `npm run npm:preversion` and verify it creates the `build/` directory.